babel.config.js

```diff
@@ -27,6 +27,7 @@ const config = {
 				exclude: [ 'transform-typeof-symbol' ],
 			},
 		],
+		'@babel/preset-typescript',
```

Comment:
Nit: the more concise `@babel/typescript` notation is also possible. Babel will infer the `preset-` and `plugin-` prefixes and suffixes.

Reply:
Cool! I didn't realize that. thanks!